commands/Factorio/getroles.js

```diff
@@ -0,0 +1,32 @@
+const { MessageEmbed } = require("discord.js");
+const Command = require("../../base/Command.js");
+
+class Linkme extends Command {
+  constructor(client) {
+    super(client, {
+      name: "getroles",
+      description: "Get in-game roles of a user",
+      usage: "<UserID/Ping>",
```

Comment:
Same as `commands/Factorio/amilinked.js`, the convention is square for required info and round for optional